pkg/cli/manifest/registermanifest.go

```diff
@@ -0,0 +1,163 @@
+/*
+You may obtain a copy of the License at
```

Comment:
Can you make sure this is the correct license header? https://github.com/radius-project/radius/pull/8120/files#diff-f0158862504865933773e70cd591585504ad7dcd16bda2dde458f967c3901fbeL2

Looks like about half of it is missing.

Reply:
my bad. i need to go over PR one more time